rbtcwrapperproxy/contracts/RBTCWrapperProxy.sol

```diff
@@ -401,20 +397,23 @@ contract RBTCWrapperProxy is ContractRegistryClient {
      *
      * @param _path         conversion path between two tokens in the network
      * @param _amount       amount to convert from, in the source token
+     * @param _recipient    the beneficiary of the conversion. If 0 address is passed, msg.sender will be considered as the recipient.
      * @param _minReturn    if the conversion results in an amount smaller than the minimum return - it is cancelled, must be greater than zero
      * 
      * @return amount of tokens received from the conversion
      */
     function convertByPath(
         IERC20Token[] memory _path,
         uint256 _amount, 
+        address _recipient,
```

Comment:
changing the ABI is a bit problematic, because third parties are using these contracts. i like that you add the recipient, but could you create another function for it? Like ConvertByPathTo(....) and the this convertByPath can just call ConvertByPathTo passing the msg.sender.

so

```
function convertByPathTo(
        IERC20Token[] memory _path,
        uint256 _amount, 
        address _recipient,
        uint256 _minReturn
    ) {
   //ALL OF THE CODE
}

function convertByPath(
        IERC20Token[] memory _path,
        uint256 _amount, 
        uint256 _minReturn
){
    convertByPathTo(_path, _amount, msg.sender, _minReturn);
}
```